### PR TITLE
Load with header code only

### PR DESCRIPTION
**Pull request by putrikarunia**
Resolves COT-44

### CODE REVIEW

File: src/handler/CompanyHandler.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import API from "../API"
+import { getAPIKeyIDFromAttr } from "../helper"
+
+const COTTER_COMPANY_CACHE_EXPIRY_KEY = "cotter_company_cache_expiry"
+const COTTER_COMPANY_CACHE_KEY = "cotter_company_info"
+
+class CompanyHandler {
+  static getInfo(): any {
+    let companyInfo = localStorage.getItem(COTTER_COMPANY_CACHE_KEY)
```
Comment (putrikarunia) [inline src/handler/CompanyHandler.ts:11]: What's the company cache key for?

**Review by putrikarunia — COMMENTED** [review]
LGTM! If the company cache key is good can merge

File: src/handler/CompanyHandler.ts (hunk 1 above)
Comment (albertpurnama) [reply src/handler/CompanyHandler.ts:11]: So need to store companyInfo in the localStorage, so if it's still not expired, we can use the one in localStorage then update it on the background.

**Review by albertpurnama — APPROVED** [review]

File: src/helper.ts [hunk 2]
```diff
@@ -169,6 +169,8 @@ export const lightOrDark = (color: any) => {
 
 export const getModalHeight = (customization: any) => {
   let modalHeight = 200;
+  console.log("customization", customization)
+  console.log("customization.captchaRequired?.toString()", customization.captchaRequired?.toString())
```
Comment (putrikarunia) [inline src/helper.ts]: remove console logs :D

**Pull request merged**